config/vufind/EPF.ini

```diff
@@ -105,6 +107,20 @@ top_rows = 2
 [ShowFacetValue]
 ;TypePublicationPubD[] = "book"
 
+; This section defines the view options available on standard search results.
+; If only one view is required, set default_view under [General] above, and comment
+; this section out.
+; The key takes the form vufindSetting_ebscoSetting -- the first part of the
+; underscore-delimited string is the view name used by VuFind (e.g. list or grid).
+; However, for EPF only list is suggested to be used. The second part is the format
+; requested from the EPF API (e.g. title, brief or detailed).
+[Views]
+; *************** is it true that we don't support grid??????????????????
+; ****************
```

Comment:
Just a note-to-self to actually test this and see how bad it looks.  Maybe under EPF it's more feasible than EDS since the journal covers would be nice visually.

Reply:
I may be misremembering, but I think the reason we disabled grid mode in EDS originally is that the views were so strongly determined by the API that it was hard to come up with a grid view that made sense. With work like #4240 making everything more flexible, there may be less of an argument against grid view than there used to be.

Reply:
That's fair.  Then if we decide to add it in the future that should certainly be a separate PR.